src/index.html

```diff
@@ -1,14 +1,59 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <title>Document</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+
+        <a href="apple" class="nav__link apple is-active">
+          apple
+        </a>
```

Comment:
1. it should be a list of links
2. such classes as apple, samsung etc are redundant
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a 
              href="apple" 
              class="nav__link is-active"
            >
              apple
            </a>
          
            .....
          </li>
        </ul>
```